alacritty_terminal/src/term/mod.rs

```diff
@@ -46,7 +46,7 @@ const TITLE_STACK_MAX_DEPTH: usize = 4096;
 const INITIAL_TABSTOPS: usize = 8;
 
 bitflags! {
-    pub struct TermMode: u32 {
+    pub struct TermMode: u64 {
```

Comment:
You can keep `u32`. The value `0b0001_0000_0000_0000_0000_0000` only needs 24 bits.

Reply:
Lol, you’re right. Not exactly sure how I miscounted that.